src/Microsoft.TestPlatform.Extensions.TrxLogger/Interfaces/ITestElement.cs

```diff
@@ -16,6 +16,7 @@ internal interface ITestElement
         TestExecId ParentExecutionId { get; set; }
         TestListCategoryId CategoryId { get; set; }
         TestCategoryItemCollection TestCategories { get; }
+        WorkitemCollection Workitems { get; set; }
```

Comment:
Please match the case with the attribute name. 

```suggestion
        WorkItemCollection WorkItems { get; set; }
```

Reply:
Updated naming